includes/controllers/class-message.php

```diff
@@ -32,33 +32,41 @@ public function __construct( $args = [] ) {
 		$args = hp\merge_arrays(
 			[
 				'routes' => [
-					'messages_resource'    => [
+					'messages_resource'     => [
 						'path' => '/messages',
 						'rest' => true,
 					],
 
-					'message_send_action'  => [
+					'message_send_action'   => [
 						'base'   => 'messages_resource',
 						'method' => 'POST',
 						'action' => [ $this, 'send_message' ],
 						'rest'   => true,
 					],
 
-					'messages_thread_page' => [
+					'messages_thread_page'  => [
 						'title'    => hivepress()->translator->get_string( 'messages' ),
 						'base'     => 'user_account_page',
 						'path'     => '/messages',
 						'redirect' => [ $this, 'redirect_messages_thread_page' ],
 						'action'   => [ $this, 'render_messages_thread_page' ],
 					],
 
-					'messages_view_page'   => [
+					'messages_view_page'    => [
 						'base'     => 'messages_thread_page',
 						'path'     => '/(?P<user_id>\d+)/?(?P<recipient_id>\d+)?',
 						'title'    => [ $this, 'get_messages_view_title' ],
 						'redirect' => [ $this, 'redirect_messages_view_page' ],
 						'action'   => [ $this, 'render_messages_view_page' ],
 					],
+
+                    'message_report_action' => [
```

Comment:
Please check WPCS setup, something clearly changed because previous commits were ok a few days ago (this is regarding spaces instead of tabs)